include/aspect/material_model/steinberger.h

```diff
@@ -305,6 +310,12 @@ namespace aspect
          */
         std::vector<std::unique_ptr<MaterialModel::MaterialUtilities::Lookup::PerplexReader> > material_lookup;
 
+        /**
+        * List of strings containing the names of the unique phases in all the material lookups
+        */
+        std::vector<std::string> unique_phase_names;
+        std::vector<std::vector<int>> unique_phase_indices;
```

Comment:
Now an unsigned int.
These have to be in the Steinberger model because unique_phase_indices is potentially a function of all of the compositional fields read into Steinberger.